Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -200,6 +203,11 @@
     m_remoteLayerTreeHost->asyncSetLayerContents(layerID, WTFMove(handle));
 }
 
+void RemoteLayerTreeDrawingAreaProxy::layerWillBeUsedForRemoteFrame(WebCore::GraphicsLayer::PlatformLayerID layerID, WebCore::FrameIdentifier frameID)
+{
+    m_frameLayerMap.set(frameID, layerID);
```

Comment:
We gotta remove them too somewhere.

Reply:
Oh, I see you `take()` them later. See my comment about member variables containing transient state.